functions/translate.js

```diff
@@ -6,6 +6,9 @@ const { Translate } = require('@google-cloud/translate').v2;
 const translate = new Translate({ projectId });
 
 const translateText = async (text, language) => {
+  if (language === 'English') language = 'en';
```

Comment:
Eventually we'll probably want to support more languages, and would probably want to pull this out into a constant outside the function. This is fine for now though